canonical-websites/www.ubuntu.com
# Update plans and pricing page

## Done
Updated the `/support/plans-and-pricing` page with the latest changes and some table highlighting.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- Check the content matches the [copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#)
- Check that the new table cell highlight is ok

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2275
